roles/mongodb/meta/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+galaxy_info:
+  description: Install MongoDB-3.2
+  author: humblearner
+  company: StackStorm
+  license: Apache
+  min_ansible_version: 2.2
+  platforms:
+    - name: Ubuntu
+      versions:
+        - trusty
+        - xenial
+    - name: CentOS/RedHat
+      versions:
+        - 6.8
```

Comment:
`6` and `7` is enough. As seen on https://github.com/jdauphant/ansible-role-nginx/blob/master/meta/main.yml